doc/source/contributing.rst

```diff
@@ -413,8 +455,11 @@ If you want to do a full clean build, do::
     python make.py clean
     python make.py html
 
-You can tell ``make.py`` to compile only a single section of the docs, greatly
-reducing the turn-around time for checking your changes.
+
+Starting with 0.13.1 you can tell ``make.py`` to compile only a single section
+of the docs, greatly reducing the turn-around time for checking your changes.
+You will be prompted to delete `.rst` files that aren't required, since the
+last committed version can always be restored from git.
```

Comment:
You can delete those two last lines, that's not true anymore. Also the `Starting with 0.13.1`, that version is from 2015, I think we can assume nobody is using it anymore.